data/net/minecraft/resources/HolderSetCodec.mapping

```diff
@@ -1,26 +1,60 @@
 CLASS net/minecraft/resources/HolderSetCodec
+	COMMENT A codec for serializing and deserializing HolderSets in Minecraft.
 	METHOD <init> (Lnet/minecraft/resources/ResourceKey;Lcom/mojang/serialization/Codec;Z)V
 		ARG 1 registryKey
 		ARG 2 elementCodec
 		ARG 3 disallowInline
 	METHOD create (Lnet/minecraft/resources/ResourceKey;Lcom/mojang/serialization/Codec;Z)Lcom/mojang/serialization/Codec;
+		COMMENT Creates a new {@linkplain HolderSetCodec} instance with the specified registry key, element codec, and inline option.
+		COMMENT <p>
+		COMMENT @return a new {@linkplain HolderSetCodec} instance.
 		ARG 0 registryKey
+			COMMENT the resource key of the registry to be used for serialization and deserialization.
 		ARG 1 holderCodec
+			COMMENT the codec for serializing and deserializing individual elements.
 		ARG 2 disallowInline
+			COMMENT if {@code true}, disallows inline definitions during decoding.
 	METHOD decode (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		COMMENT Decodes the dynamic value using the specified dynamic ops into a data result containing a HolderSet and the remaining value.
+		COMMENT <p>
+		COMMENT @return a data result containing the decoded {@linkplain HolderSet} and the remaining value or an error if the decoding fails.
 		ARG 1 ops
+			COMMENT the dynamic ops used for decoding.
 		ARG 2 input
+			COMMENT the dynamic value to be decoded.
 	METHOD decodeWithoutRegistry (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		COMMENT Decodes the provided dynamic value using the specified dynamic ops without registry context into a data result containing a HolderSet and the remaining dynamic value.
+		COMMENT <p>
+		COMMENT @return a data result containing the decoded HolderSet and the remaining dynamic value or an error if the decoding fails.
 		ARG 1 ops
+			COMMENT the dynamic ops used for decoding.
 		ARG 2 input
+			COMMENT the dynamic value to be decoded.
 	METHOD encode (Lnet/minecraft/core/HolderSet;Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		COMMENT Encodes the provided {@linkplain HolderSet} into a data result using the specified dynamic ops.
+		COMMENT <p>
+		COMMENT @return a data result containing the encoded value or an error if the encoding fails.
 		ARG 1 input
+			COMMENT the {@linkplain HolderSet} to be encoded.
 		ARG 2 ops
+			COMMENT the dynamic ops used for encoding.
 		ARG 3 prefix
+			COMMENT the dynamic value to be encoded into.
```

Comment:
I'm not sure if this is *wrong* per say but I don't think it's *right*, either. Honestly, it feels very weird to be documenting a single override of `Codec.encode` in parchment, and I would tend to suggest deferring to, e.g. [this](https://kvverti.github.io/Documented-DataFixerUpper/snapshot/com/mojang/serialization/Encoder.html#encode-A-com.mojang.serialization.DynamicOps-T-).

It's also going to be really weird if we have, for every `encode` and `decode` overriding `Codec`, either has a comment, doesn't have a comment, or has a different comment (because I do not trust everyone do document these identically.